src/olympia/files/utils.py

```diff
@@ -446,7 +446,7 @@ def extract_xpi(xpi, path, expand=False):
     contents. If you have 'foo.jar', that contains 'some-image.jpg', then
     it will create a folder, foo.jar, with an image inside.
     """
-    expand_whitelist = ['.jar', '.xpi']
+    expand_whitelist = ['.jar', '.xpi', '.zip']
```

Comment:
regarding tests, it appears there aren't any specifically for `extract_expi` which is a shame, we do have some indirect tests in https://github.com/mozilla/addons-server/blob/master/src/olympia/files/tests/test_helpers.py#L63 (and many more following, look for `self.view.extract()` and folling assertions) which actually tests quite a lot of it so maybe it's the best palce to start adding some tests or reading into how things work.